src/wrappers/themis/rust/libthemis-sys/bindgen.sh

```diff
@@ -24,6 +24,16 @@ set -e -o pipefail
 # Bindgen sees Soter as well as some system libraries, we don't need that.
 WHITELIST="(THEMIS|themis|secure_(comparator|session)|STATE)_.*"
 
+EXTRA_FLAGS=
+
+# This flag was deprecated and removed in 0.65.0 [1].
+# However, older bindgen versions still require it.
+#
+# [1]: https://github.com/rust-lang/rust-bindgen/pull/2408
+if [[ $(bindgen | grep -- '--size_t-is-usize') ]]; then
```

Comment:
```suggestion
if bindgen | grep -q -- '--size_t-is-usize'; then
```
I'd suggest to make it a bit simpler. Am I missing something and `test` construction has been added intentionally?

Reply:
Well, I'm not advanced `sh` user, so I thought that this construction will not work with the `set -eo`. But I checked and I was wrong. Thank you! :)